lib/rubocop/cop/style/invertible_unless_condition.rb

```diff
@@ -88,6 +91,47 @@ def inheritance_check?(node)
             (argument.const_type? && argument.short_name.to_s.upcase != argument.short_name.to_s)
         end
 
+        def preferred_condition(node)
+          case node.type
+          when :begin
+            if node.parenthesized_call?
+              "(#{preferred_condition(node.children.first)})"
+            else # TODO: What about other kinds of begin nodes?
```

Comment:
I forgot about this comment, but maybe it still applies.

Do we care about things like this?

```ruby
if begin; dangerous_condition; rescue => SomeError; fallback_condition; end;
  # ...
end
```

Reply:
Okay, so I tried

```ruby
foo unless begin x != y end
foo unless (begin x != y end)
```

but neither detected an offense, so I'll just assume the parenthesized case is the only one we care about, and simplify the implementation.